SalonCheckIn-frontend/src/App.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Redirect, Route, Switch } from 'react-router-dom/cjs/react-router-dom.min';
+import "../node_modules/bootstrap/dist/css/bootstrap.min.css";
+import "../node_modules/bootstrap/dist/js/bootstrap.bundle";
+import './index.css';
+import Home from './Home';
+import Registration from './Registration';
+import Login from './Login';
+import ShopProfile from './ShopProfile';
+import CustomerProfile from './CustomerProfile';
+import Navbar from './Navbar';
+import Footer from './Footer';
+import RegisterCustomer from './RegisterCustomer';
+import RegisterSalon from './RegisterSalon';
+import ShopList from './ShopList';
```

Comment:
make different directory with appropriate name for each related component.

Reply:
okay,darshik